flac.md

```diff
@@ -363,13 +363,25 @@ Note that the Vorbis comment as used in Vorbis allows for on the order of 2\^64
 
 ### Standard field names
 
-Only one standard field name is defined; the channel mask field, in (#channel-mask). In general, most FLAC playback devices and software recognize the following field names:
+Only one standard field name is defined: the channel mask field, in (#channel-mask). No other field names are defined because the applicability of any field name is strongly tied to the content it is associated with. For example, field names useful for describing files that contain a single work of music would be unusable when labeling archived broadcasts, recordings of any kind, or a collection of music works. Even when describing a single work of music, different conventions exist depending on the kind of music: orchestral music differs from music by solo artists or bands.
```

Comment:
```suggestion
Only one standard field name is defined: the channel mask field, in (#channel-mask). Other field names are free-form field names with their meaning being outside the scope of this specification because the applicability of any field name is strongly tied to the content it is associated with. For example, field names useful for describing files that contain a single work of music would be unusable when labeling archived broadcasts, recordings of any kind, or a collection of music works. Even when describing a single work of music, different conventions exist depending on the kind of music: orchestral music differs from music by solo artists or bands.
```

Or

```suggestion
Only one standard field name is defined: the channel mask field, in (#channel-mask). Other field names are free-form field names with their meaning being outside the scope of this specification.
```

(I am not sure that there is a need of more details here, it is just outside of this spec)

Reply:
In last call review and IESG reviews the term 'hand-waiving' was used a few times, I believe at least once in this context. I think there is a need to explain why no tags are formalized: not because we are too "lazy" to define this but because there is a real reason why defining a one-size-fits-all approach for field names is impossible: use cases are too different. It is not out-of-scope per se, if it was straightforward I would have included it.